src/main/java/org/betonquest/betonquest/database/MySQL.java

```diff
@@ -21,6 +21,7 @@
  */
 @SuppressWarnings({"PMD.CommentRequired", "PMD.AvoidDuplicateLiterals"})
 public class MySQL extends Database {
+
```

Comment:
Sus new empty line
Unfortunately we do not have a check style rule for this and no decision what we want

Reply:
Fixed, was caused by a constant that I created for a slightly different attempt.